app/views/pages/home.html.erb

```diff
@@ -4,39 +4,36 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
           <h1 class="govuk-heading-l">Get an adviser</h1>
-          <p>This service is for those who want to become a teacher in England.</p>
-          <p>To use this service you need to have one of the following:</p>
+
+          <p>If you want to become a teacher or return to teaching in England, a Teacher Training Adviser can guide you through the process with free one-to-one support.
```

Comment:
I'd recommend indenting the content here _or_ bringing the trailing `</p>` up onto the same line.